tests/admin_scripts/tests.py

```diff
@@ -2359,6 +2359,11 @@ def test_invalid_choice_db_option(self):
             "Error: argument --database: invalid choice: "
             "'deflaut' (choose from 'default', 'other')"
         )
+        if PY314:
+            expected_error = (
+                "Error: argument --database: invalid choice: "
+                "'deflaut' (choose from default, other)"
+            )
```

Comment:
I would use `if PY314: ... else: ...` pattern (in both cases) to make it clearer what is expected where, e.g.
```python
if PY314:
    expected_error = (
        "Error: argument --database: invalid choice: 'deflaut' "
        "(choose from default, other)"
    )
else:
    expected_error = (
        "Error: argument --database: invalid choice: 'deflaut' "
        "(choose from 'default', 'other')"
    )
```